resources/js/Pages/Home.vue

```diff
@@ -42,18 +43,51 @@
             user: Object,
         },
         methods: {
+            checkEmpty() {
+                if( !this.form.itemsInput ) {
+                    this.error = {
+                        type: 'warning',
+                        message: 'item-form-error-message-empty'
+                    };
+                    return false;
+                }
+                return true;
+            },
+            validate() {
+                let valid = true;
+                const lines = this.form.itemsInput.split( '\n' );
+                for ( let line of lines ) {
```

Comment:
There are some spiffy JS higher order functions for checks like this, namely: [`Array.prototype.every()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every). Of course, this is optional, and the implementers choice :smile: